src/components/Sidebar/Sidebar.css

```diff
@@ -37,6 +37,7 @@ aside .profile {
   font-weight: bold;
   display: flex;
   align-items: center;
+  justify-content: space-between;
 }
 
 aside .username {
```

Comment:
don't target the elements directly

Reply:
Okay, I will keep this in mind. Thank you.